### ISSUE TITLE
[Bug] Creating Backups fails

### ISSUE THREAD

**Issue by Dschogo** [InvoiceShelf/docker#4]
**Describe the bug**
Creating a backup results in an error `Undefiend constant: Spatie\Backup\Tasks\Backup\SIGINT`.

![image](https://github.com/InvoiceShelf/InvoiceShelf/assets/36862419/0088091a-7dbf-49d1-b023-d444639270a6)

local_public / local_private nor backup type changes this behavior.

**Expected behavior**
Create a backup without error.

**Additional info**
How should a backup be restored? Didn't see anything during setup/menu option. Enter docker (for me) and execute restore directly on the database via cli?

**Please complete the following information:**
- InvoiceShelf version: docker-compose (invoiceshelf/invoiceshelf:latest) - 1.1.0
- PHP version: bundled
- Database type and version: bundled

**Optional info**
- OS:  docker host: debian arm64
- Browser: Firefox

<details>
  <summary>docker-compose.yml</summary>

```yml
#-------------------------------------------
#  Docker Compose
#  - Darko Gjorgjijoski
# Repo : https://github.com/InvoiceShelf/docker
#-------------------------------------------

version: '3'

services:
   invoiceshelf_db:
     container_name: invoiceshelf_db
     image: mariadb:10
     environment:
       - MYSQL_ROOT_PASSWORD=randompw
       - MYSQL_DATABASE=invoiceshelf
       - MYSQL_USER=invoiceshelf
       - MYSQL_PASSWORD=randompw
     expose: 
       - 3306
     volumes:
       - mysql:/var/lib/mysql
     networks:
       - invoiceshelf
     restart: unless-stopped

   invoiceshelf:
     image: invoiceshelf/invoiceshelf
     container_name: invoiceshelf
     ports:
       - 90:80
     volumes:
       - ./invoiceshelf/data:/data
     networks:
       - invoiceshelf
     environment:
       - PUID=1000
       - PGID=1000
       - USER=invoiceshelf
       - PHP_TZ=UTC
       - TIMEZONE=UTC
       - DB_CONNECTION=mysql
       - DB_HOST=invoiceshelf_db
       - DB_PORT=3306
       - DB_DATABASE=invoiceshelf
       - DB_USERNAME=invoiceshelf
       - DB_PASSWORD=randompw
       - STARTUP_DELAY=0
     restart: unless-stopped
     depends_on:
       - invoiceshelf_db

networks:
   invoiceshelf:

volumes:
    mysql:
```
</details>

**Comment by sebarnd** [comment]
Hi,

according to https://github.com/spatie/laravel-backup/issues/1445 this is due to the need of the pcntl php extension which is missing in the docker image. I'm afraid the most easy way to solve that would be to switch to the official php-fpm docker image?! @gdarko what do you think?

**Comment by Dschogo** [comment]
Ah good find!

Then something like this in the Dockerfile should fix it, can test it tomorrow.
```
RUN docker-php-ext-configure pcntl --enable-pcntl \
  && docker-php-ext-install pcntl;
```

**Comment by Dschogo** [comment]
Update: doesnt work that easy, i've managed to include pcntl by installing it via pecl/pear but same result. 

Strangely `php -i | grep pcntl` results even in the current image to

```
root@10305fca2c7c:/var/www/html/InvoiceShelf# php -i | grep pcntl
pcntl
pcntl support => enabled
```
so it's already there but can't be used/ found? Maybe using a base php image is the only easier way.

**Comment by nielsdrost7** [comment]
> Maybe using a base php image is the only easier way.
Or copy it from the baseImage into your own produced image?

**Comment by gdarko** [comment]
The issue will be resolved once v1.2.0 is out, soon.

**Comment by gdarko** [comment]
The issue has been resolved as of 1.2.0+ version.

Best Regards,
Darko
